src/components/Messages/MessagePreview.jsx

```diff
@@ -1,14 +1,25 @@
 // React imports
 import React, { useState } from 'react';
 // Styling Imports
-import styled from 'styled-components';
+import { StyledDate, StyledHeader, StyledPreview } from './MessageStyles';
+
+/**
+ * messageFolderProps is an object that stores the props for the MessageFolder
+ * component
+ *
+ * @typedef messageFolderProps
+ * @type {object}
+ * @property {string} message - The content of the message sent
+ * @memberof typedefs
+ */
 
 /**
  * Message Preview Component - Component that displays message previews from
  * user's Inbox container
  *
  * @memberof Inbox
  * @name MessagePreview
+ * @param {messageFolderProps} Props - Component props for MessagePreview
  */
 
```

Comment:
I don't know if it's just my setup, but these blank between the JSDoc description and the function prevent my JSDoc validation from running. Can we remove them?

Reply:
I can do that.